homeassistant/components/skybell/sensor.py

```diff
@@ -27,56 +24,44 @@
         icon="mdi:bell-ring",
     ),
 )
-MONITORED_CONDITIONS: list[str] = [desc.key for desc in SENSOR_TYPES]
-
 
+# Deprecated in Home Assistant 2022.5
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Optional(
-            CONF_ENTITY_NAMESPACE, default=DEFAULT_ENTITY_NAMESPACE
-        ): cv.string,
+        vol.Optional(CONF_ENTITY_NAMESPACE, default=DOMAIN): cv.string,
         vol.Required(CONF_MONITORED_CONDITIONS, default=[]): vol.All(
-            cv.ensure_list, [vol.In(MONITORED_CONDITIONS)]
+            cv.ensure_list, [vol.In(SENSOR_TYPES)]
         ),
     }
 )
 
```

Comment:
Please leave the schema as-is since its going away.  

We don't allow schema changes for the old platforms per ADR0007 https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md

I realize this isn't a functional change, but it makes it more difficult to review